actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -96,11 +96,11 @@ def initialize(response, buf)
       end
 
       def body
-        @str_body ||= begin
-                        buf = ''
-                        each { |chunk| buf << chunk }
-                        buf
-                      end
+        begin
```

Comment:
@sgrif tests passed locally with the memoization added back. Travis should pick that change up momentarily.